src/concept/type/ThingType.jl

```diff
@@ -24,105 +27,130 @@ label(t::AbstractThingType) = t.label
 # Remote functions
 # ------------------------------------------------------------------------
 
-function set_supertype(r::RemoteConcept{C,T}) where {
-    C <: AbstractThingType, T <: AbstractCoreTransaction
-}
-    req = ThingTypeRequestBuilder.set_supertype_req(r.concept.label)
-    res = execute(r.transaction, req)
+function set_supertype(r::RemoteConcept{C,T},
+                        super_type::AbstractThingType) where
+                        {C <: AbstractThingType, T <: AbstractCoreTransaction}
+
+    req = ThingTypeRequestBuilder.set_supertype_req(r.concept.label, proto(super_type))
+    execute(r.transaction, req)
 end
 
-function get_supertype(r::RemoteConcept{C,T}) where {
-    C <: AbstractThingType, T <: AbstractCoreTransaction
-}
+function get_supertype(r::RemoteConcept{C,T}) where {C <: AbstractType,T <: AbstractCoreTransaction}
```

Comment:
Are you using a formatter? Blue style goes with 92 characters max and it should break...

Reply:
No. I didn't know which formatter is suitable for an automatic formatting.